composer.json

```diff
@@ -31,24 +31,24 @@
         "php": ">=7.0.0 <7.3",
         "helhum/typo3-console-plugin": "^2.0.2",
 
-        "typo3/cms-backend": "^8.7.10 || >=9.1.0 <9.5",
-        "typo3/cms-core": "^8.7.10 || >=9.1.0 <9.5",
-        "typo3/cms-extbase": "^8.7.10 || >=9.1.0 <9.5",
-        "typo3/cms-extensionmanager": "^8.7.10 || >=9.1.0 <9.5",
-        "typo3/cms-fluid": "^8.7.10 || >=9.1.0 <9.5",
-        "typo3/cms-frontend": "^8.7.10 || >=9.1.0 <9.5",
-        "typo3/cms-install": "^8.7.10 || >=9.1.0 <9.5",
+        "typo3/cms-backend": "^8.7.10 || >=9.1.0 <9.6",
```

Comment:
Since there won't be a minor release for 9.x anymore wouldn't `^9.1` make more sense now?

Reply:
I'd like to proceed with a release compatible with 9.5.0 with the least possible changes.

After that is done for today, I will release a new minor version, which removes support for the sprint releases entirely and only allow `^8.7.10 || ^9.5`